2.crud_queries/tasks.md

```diff
@@ -1,38 +1,82 @@
 1. Create a database named `blog`.
 
+use blog
+
 2. Create a collection called 'articles'.
 
+db.createCollection('articles')
+
 3. Insert multiple documents(at least 3) into articles. It should have fields
 
+db.articles.insertMany([{title: 'GOT-1', description: 'A', author: {name: 'A', age: 65}, tags: ['A']},
+{title: 'GOT-2', description: 'B', author: {name: 'B',  age: 66}, tags: ['B']},
+ {title: 'GOT-3', description: 'C', author: {name: 'C', age: 67}, tags: ['C']}])
+
 4. Find all the articles using `db.COLLECTION_NAME.find()`
 
+db.articles.find()
+
 5. Find a document using _id field.
 
+db.articles.find({ "_id" : ObjectId("5d9cb039b43283ca02645914")})
+
+
 6. Find documents using title and author's name field.
 
+db.articles.find({title: 'GOT-1'})
+db.articles.find({title: 'GOT-2'})
+db.articles.find({title: 'GOT-3'})
+
 7. Find document using a specific tag.
 
+db.articles.find({'author.name': 'A'})
+db.articles.find({tags: 'B'})
+
 8. Update title of a document using its _id field.
 
+db.articles.update({_id: ObjectId("5d5c280962da9e9b5f997eb0")}, {$set: {title: 'GOT-1'}})
+
 9. Update a author's name using article's title.
 
+db.articles.update({title: 'GOT-3'}, {$set: {'author.name': 'CC'}})
+
 10. rename details field to description from articles collection. 
 
+db.articles.updateMany({}, {$rename: {'details':'description'}})
+
 11. Add additional tag in a specific document.
 
```

Comment:
Since tags field is storing array of strings. You can use $push operator to add specific tag into tags array.
```js
db.articles.update({title: 'GOT-3'}, {$set: {$push: {tags: "node"})
```